frontend/src/components/Diff/Diff.module.scss

```diff
@@ -133,3 +128,12 @@
 .rotation6 { color: lightcyan; }
 .rotation7 { color: lightgreen; }
 .rotation8 { color: grey; }
+
+.highlighted {
+    color: white;
+    background-color: #aa8b00;
+}
+
+.highlightable {
+    cursor: pointer;
```

Comment:
I also experimented with this and felt it a bit distracting. Though it's true that it does add discoverability

Reply:
I definitely think the discoverability is worth it